### PR TITLE
[IOPID-911] : Review Footer

### PR DESCRIPTION
**Pull request by ClaudioNTT**
## Short Description
The goal is to align the footer with the one specified in the JIRA [ticket](https://pagopa.atlassian.net/jira/software/c/projects/IOPID/boards/457?modal=detail&selectedIssue=IOPID-911). 

## List of Proposed Changes in this Pull Request
- Footer Text & Links Review
- Added ENV VAR for footer products
## How to Test
Run the project and verify two scenarios:

**Not Logged In**

Ensure that the footer matches the one found [here](https://notifichedigitali.pagopa.it/).
Note: The product list, as per documentation, is retrieved by the link specified in the productsJsonUrl prop (please refer to [this link](https://main--633c31eff9fe385398ada426.chromatic.com/?path=/story/components-footer-wip--pre-login&globals=backgrounds.grid:false)). Unfortunately, in test mode, there is a CORS error when using the correct link https://notifichedigitali.pagopa.it/static/product.json, but it works fine if we use the other one https://dev.selfcare.pagopa.it/assets/products.json.

**Logged In**

Ensure that the footer matches the provided picture.
image

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1,25 +1,27 @@
...
+NEXT_PUBLIC_ANALYTICS_MOCK=true # FIXME: JIRA TICKET 862
+NEXT_PUBLIC_ANALYTICS_TOKEN="" # FIXME: JIRA TICKET 862
+NEXT_PUBLIC_ANALYTICS_API_HOST="" # FIXME: JIRA TICKET 862
+NEXT_PUBLIC_ANALYTICS_PERSISTENCE="" # FIXME: JIRA TICKET 862
+NEXT_PUBLIC_ANALYTICS_LOG_IP=true
+NEXT_PUBLIC_ANALYTICS_DEBUG=true
+
+NEXT_PUBLIC_SPID_TEST_ENV_ENABLED=true
+
+NEXT_PUBLIC_FOOTER_PRODUCT_LIST=https://dev.selfcare.pagopa.it/assets/products.json
```
Comment (shadowsheep1) [inline .env.example:27]: Why I see all this changes? Maybe different IDEs encoding of text files?

File: .env.local [hunk 2]
```diff
@@ -1,25 +1,27 @@
...
+NEXT_PUBLIC_API_FETCH_MAX_RETRY=5
+
+NEXT_PUBLIC_FEATURE_FLAG=false
+NEXT_PUBLIC_SPID_CIE_ENTITY_ID=xx_servizicie
+
+NEXT_PUBLIC_ANALYTICS_ENABLE=true
+NEXT_PUBLIC_ANALYTICS_MOCK=true # FIXME: JIRA TICKET 862
+NEXT_PUBLIC_ANALYTICS_TOKEN= # FIXME: JIRA TICKET 862
+NEXT_PUBLIC_ANALYTICS_API_HOST= # FIXME: JIRA TICKET 862
+NEXT_PUBLIC_ANALYTICS_PERSISTENCE= # FIXME: JIRA TICKET 862
```
Comment (shadowsheep1) [inline .env.local:21]: Why here is there any missing string like in other `.env` files?

File: .env.production [hunk 3]
```diff
@@ -1,25 +1,27 @@
...
+NEXT_PUBLIC_ANALYTICS_MOCK=true # FIXME: JIRA TICKET 862
+NEXT_PUBLIC_ANALYTICS_TOKEN="" # FIXME: JIRA TICKET 862
+NEXT_PUBLIC_ANALYTICS_API_HOST="" # FIXME: JIRA TICKET 862
+NEXT_PUBLIC_ANALYTICS_PERSISTENCE="" # FIXME: JIRA TICKET 862
+NEXT_PUBLIC_ANALYTICS_LOG_IP=true
+NEXT_PUBLIC_ANALYTICS_DEBUG=true
+
+NEXT_PUBLIC_SPID_TEST_ENV_ENABLED=false
+
+NEXT_PUBLIC_FOOTER_PRODUCT_LIST=https://dev.selfcare.pagopa.it/assets/products.json
```
Comment (shadowsheep1) [inline .env.production:27]: as above

File: .env.test.local [hunk 4]
```diff
@@ -1,25 +1,27 @@
...
+NEXT_PUBLIC_ANALYTICS_MOCK=true # FIXME: JIRA TICKET 862
+NEXT_PUBLIC_ANALYTICS_TOKEN="" # FIXME: JIRA TICKET 862
+NEXT_PUBLIC_ANALYTICS_API_HOST="" # FIXME: JIRA TICKET 862
+NEXT_PUBLIC_ANALYTICS_PERSISTENCE="" # FIXME: JIRA TICKET 862
+NEXT_PUBLIC_ANALYTICS_LOG_IP=true
+NEXT_PUBLIC_ANALYTICS_DEBUG=true
+
+NEXT_PUBLIC_SPID_TEST_ENV_ENABLED=true
+
+NEXT_PUBLIC_FOOTER_PRODUCT_LIST=https://dev.selfcare.pagopa.it/assets/products.json
```
Comment (shadowsheep1) [inline .env.test.local:27]: as above

**Review by shadowsheep1 — CHANGES_REQUESTED** [review]

File: .env.example (hunk 1 above)
Comment (ClaudioNTT) [reply .env.example:27]: During PC env reinstallation I forgot to set LF instead of CRLF inside VS code

**Pull request merged**